integrationservertest/steps.go

```diff
@@ -175,6 +187,13 @@ func (i ingestStep) Step(t *testing.T, ctx context.Context, e *testStepEnv) {
 
 	t.Logf("------ ingest in %s------", e.currentVersion())
 	err := e.gen.RunBlockingWait(ctx, e.currentVersion(), e.integrations)
+	if err != nil {
```

Comment:
Why only check for panic logs if there was ab error here? Couldn't it also happen that some data were ingested but a panic still happened somewhere?